Python/pywarpx/picmi.py

```diff
@@ -737,10 +743,12 @@ def initialize_inputs(self):
         self.solver.initialize_inputs()
 
         for i in range(len(self.species)):
-            self.species[i].initialize_inputs(self.layouts[i],
-                                              self.initialize_self_fields[i],
-                                              self.injection_plane_positions[i],
-                                              self.injection_plane_normal_vectors[i])
+            self.species[i].initialize_inputs(
+                self.layouts[i],
+                False if self.initialize_self_fields[i] is None else True,
```

Comment:
Why is this change made? None is a valid value for the initialize_self_fields argument.

Reply:
Thanks for pointing this out. I don't actually remember what spurred this change but it is certainly not needed for anything in this PR so I changed it back.